invenio_rdm_pure/views.py

```diff
@@ -44,3 +45,11 @@ def index3():
     command += f"owner --identifier='externalId' --identifierValue='{externalId}'"
     os.system(command)
     return "Task successfully completed"
+
+
+@blueprint.route("/test")
+def test():
+    """Tests addon functionalities."""
+    test = Testing()
+    test.run_tests()
+    return "Tests finished successfully."
```

Comment:
Do we still need this?

Reply:
Good point. 8584fcce0c7765c73a170a41e350bc70151674d3 removes these lines.